src/variables.ts

```diff
@@ -0,0 +1,672 @@
+      { label: 'Bus E MeterF1', name: 'bus_e_meterf1' },
+      { label: 'Bus E MeterFz', name: 'bus_e_meterf2' },
+      { label: 'Bus E Mute', name: 'bus_e_mute' },
+      { label: 'Bus E Solo', name: 'bus_e_solo' },
+      { label: 'Bus F Volume', name: 'bus_f_volume' },
+      { label: 'Bus F dB', name: 'bus_f_volume_db' },
+      { label: 'Bus F Volume Linear', name: 'bus_f_volume_linear' },
+      { label: 'Bus F MeterF1', name: 'bus_f_meterf1' },
+      { label: 'Bus F MeterFz', name: 'bus_f_meterf2' },
+      { label: 'Bus F Mute', name: 'bus_f_mute' },
+      { label: 'Bus F Solo', name: 'bus_f_solo' },
+      { label: 'Bus G Volume', name: 'bus_g_volume' },
+      { label: 'Bus G dB', name: 'bus_g_volume_db' },
+      { label: 'Bus G Volume Linear', name: 'bus_g_volume_linear' },
+      { label: 'Bus G MeterF1', name: 'bus_g_meterf1' },
+      { label: 'Bus G MeterFz', name: 'bus_g_meterf2' },
+      { label: 'Bus G Mute', name: 'bus_g_mute' },
+      { label: 'Bus G Solo', name: 'bus_g_solo' },
+
+      // Overlay
+      { label: 'Overlay 1 Input Short Title', name: 'overlay_1_input_name' },
+      { label: 'Overlay 1 Input Number', name: 'overlay_1_input' },
+      { label: 'Overlay 1 Active PGM', name: 'overlay_1_pgm' },
+      { label: 'Overlay 1 Active PRV', name: 'overlay_1_prv' },
+      { label: 'Overlay 2 Input Short Title', name: 'overlay_2_input_name' },
+      { label: 'Overlay 2 Input Number', name: 'overlay_2_input' },
+      { label: 'Overlay 2 Active PGM', name: 'overlay_2_pgm' },
+      { label: 'Overlay 2 Active PRV', name: 'overlay_2_prv' },
+      { label: 'Overlay 3 Input Short Title', name: 'overlay_3_input_name' },
+      { label: 'Overlay 3 Input Number', name: 'overlay_3_input' },
+      { label: 'Overlay 3 Active PGM', name: 'overlay_3_pgm' },
+      { label: 'Overlay 3 Active PRV', name: 'overlay_3_prv' },
+      { label: 'Overlay 4 Input Short Title', name: 'overlay_4_input_name' },
+      { label: 'Overlay 4 Input Number', name: 'overlay_4_input' },
+      { label: 'Overlay 4 Active PGM', name: 'overlay_4_pgm' },
+      { label: 'Overlay 4 Active PRV', name: 'overlay_4_prv' },
+
+      // Dyanmic Inputs/Values
+      { label: 'Dynamic Input 1', name: 'dynamic_input_1' },
+      { label: 'Dynamic Input 2', name: 'dynamic_input_2' },
+      { label: 'Dynamic Input 3', name: 'dynamic_input_3' },
+      { label: 'Dynamic Input 4', name: 'dynamic_input_4' },
+      { label: 'Dynamic Value 1', name: 'dynamic_value_1' },
+      { label: 'Dynamic Value 2', name: 'dynamic_value_2' },
+      { label: 'Dynamic Value 3', name: 'dynamic_value_3' },
+      { label: 'Dynamic Value 4', name: 'dynamic_value_4' },
+
+      // Layers
+      { label: 'Layer Routing Input', name: 'layer_routing_input' },
+      { label: 'Layer Routing Layer', name: 'layer_routing_layer' },
+    ])
+
+    // Inputs
+    const inputNumberVariables: Set<InstanceVariableDefinition> = new Set([])
```

Comment:
You can also write this as `const inputNumberVariables = new Set<InstanceVariableDefinition>()`